### ISSUE TITLE
Works only with TU Darmstadt fork of dynamixel workbench

### ISSUE THREAD

**Issue by SammyRamone** [tu-darmstadt-ros-pkg/dynamixel_workbench_ros_control#1]
Hi,
first thanks for the code, you spared me some work :)
The thing is, this works only with your forked version of the Dynamixel Workbench. Are you planning on doing a pull request of it in the future?

**Comment by astumpf** [comment]
Hi,
yeah, I guess we have to sync all our forks in the near future. However, the current state of this project should be considered as early alpha as some implemented features aren't well tested so far. Expect major updates and fixes in the future. Feel free to submit PR as well :-).

**Comment by SammyRamone** [comment]
Sure the ROS control part should be considered as alpha. But I think the fork of the workbench can be quiet easily be merged, its only adding new methods. This should not be able to break anything. There is only a problem with code duplication in the new syncWritePosition, syncWriteVelocity and syncWriteEffort methods. If you don't mind I can fork your fork, try to improve this and do a PR on the Robotis git.

**Comment by astumpf** [comment]
Sure, we appreciate any help! Please mind to credit Martin Oehler when doing the PR.

**Comment by SammyRamone** [comment]
I opened a pull request:
https://github.com/ROBOTIS-GIT/dynamixel-workbench/pull/97

**Comment by SammyRamone** [comment]
So it seems, that there is a develop branch on the Robotis Git, which is far ahead of the master, so your code will not be merged. 
I wrote a new version of your ros control (https://github.com/bit-bots/dynamixel_workbench_ros_control) which is based on this develop branch.
I created a new pull request (https://github.com/ROBOTIS-GIT/dynamixel-workbench/pull/105) to get the feature of reading multiple registers when doing a sync read, so currently my version is also not compatible with the version from robotis.
Any way, if you're interested have a look at my version, maybe it is useful for you.
